app/views/merchant_details/edit_merchant_details.njk

```diff
@@ -56,39 +56,29 @@
     <p class="govuk-body" id="merchant-details-info">Direct Debit requires the details of the organisation taking payment to be shown to the payer. This information is
       displayed in emails and on payment pages.</p>
   {% elif has_card_gateway_account %}
-    <p class="govuk-body" id="merchant-details-info">Payment card schemes require the details of the organisation taking payment to be shown on payment pages.</p>
+    <p class="govuk-hint" id="merchant-details-info">Payment card schemes require the details of the organisation taking payment to be shown on payment pages.</p>
   {% endif %}
-
+  <br>
```

Comment:
We generally use spacing classes to control the amount of vertical space, rather than line breaks.

https://design-system.service.gov.uk/styles/spacing/

Reply:
👍  Updated